dbscripts/xml/upgrade/ms_sql_support_update.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * ms_sql_support_update.xml
```

Comment:
Better to specify the full path to the filename -- and if you don't mind, please use a code header like the other files contain (copyright, license, etc).